project/newEvaluation.html

```diff
@@ -0,0 +1,474 @@
+  </header>
+
+  <hr>
+
+  <form class="center">
+
+      <section>
+        <input type="text" placeholder="Candidate">
+        <input type="text" placeholder="Interviewer">
+        <input type="date" style="border: none">
+      </section>
+
+      <section>
+        <h2>Technical Level</h2>
+        <table style="margin-top: 0%">
+          <tr>
+            <th>Trainee</th>
+            <th>Junior</th>
+            <th>Middle</th>
+            <th>Senior</th>
+          </tr>
+          <tr class="border-lists">
+            <td>
+              <input type="radio" id="T"/>
+            </td>
+            <td>
+              <input type="radio" id="J1"/>
+              <input type="radio" id="J2"/>
+              <input type="radio" id="J3"/>
+            </td>
+            <td>
+              <input type="radio" id="M1"/>
+              <input type="radio" id="M2"/>
+              <input type="radio" id="M3"/>
+            </td>
+            <td>
+              <input type="radio" id="S1"/>
+              <input type="radio" id="S2"/>
+              <input type="radio" id="S3"/>
+            </td>
+          </tr>
+        </table>
+      </section>
+
+
+      <section>
+        <h2>Should the candidate be hired?</h2>
+        <textarea rows="6" placeholder="The type of project that is suitable for the candidate
+Is guidance required for the candidate" reqired></textarea>
+      </section>
+
+      <section>
+        <h2>General Impression</h2>
+        <textarea rows="6" required placeholder="*required" impressions"></textarea>
+      </section>
+
+      <section>
+        <h2>Workflow, Leadership & Soft Skills</h2>
+        <textarea rows="6" required placeholder="Describe the environment in which the candidate works.
+    Describe any guidance or management experience" workflow"></textarea>
+      </section>
+
+      <fieldset>
+        <legend>OOP, Design Patterns</legend>
+        <ul>
+          <li>
+            <label for="classes">Classes</label>
+            <select name="classes" id="classes">
+              <option value="0">0</option>
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+            </select>
+            <label for="exception">Exception Handling</label>
+            <select name="exception" id="exception">
+              <option value="0">0</option>
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+            </select>
+            <label for="vcs">Version Control</label>
+            <select name="vcs" id="vcs">
+              <option value="0">0</option>
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+            </select>
+          </li>
+          <li>
+            <label for="access">Access Modifiers</label>
```

Comment:
What is this `for` attribute and what does it do? Do you know of any other alternative that does the same thing?